ubuntu20.04/generate-control-file

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -e
+
+
+function get-arch {
+    if [[ "$HOST_ARCH" =~ ^(broadwell|haswell|skylake|x86_64)$ ]];
+    then
+        echo "amd64"
+    elif [[ "$HOST_ARCH" =~ ^(wasm32|wasm64)$ ]];
+    then
+        echo "wasi"
+    elif [[ "$HOST_ARCH" =~ ^(armv8-a)$ ]];
+    then
+        echo "aarch64"
+    fi
+}
+
+function generate-control-file {
+    PACKAGE_NAME=$1
+    DEPENDENCIES=$2
+    SYSTEM_ARCH="$(get-arch)"
+    SECTION=${SECTION:-"devel"}
+    SOURCE_PACKAGE_VERSION=$3
+    GIT_URL=https://github.com/${PACKAGE_BASE_NAME}
+    # Template definition
+    cat << EOF
+Source: ${SOURCE_PACKAGE_NAME}
+Section: ${SECTION}
+Priority: important
+Maintainer: Val Verde Inc. <openvalverde@gmail.com>
```

Comment:
This field should be passed in as well.

Reply:
removed this file to address deb generation as a separate PR